packages/docs/src/page-configs/ui-elements/data-table/examples/CRUD.vue

```diff
@@ -138,11 +139,28 @@ export default defineComponent({
 </script>
 
 <style lang="scss" scoped>
-  .table-example--slot {
-    th {
-      padding-top: 5px;
-      padding-bottom: 5px;
-      vertical-align: middle;
+  .table-example {
+    &__slot {
+      th {
+        padding-top: 5px;
+        padding-bottom: 5px;
+        vertical-align: middle;
+      }
+
+      .va-input {
+        --va-form-element-default-width: 80px;
```

Comment:
```suggestion
        --va-form-element-default-width: 0;
```

`va-form-element-default-width` can be set inside `va-data-table` component instead.